src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections.Generic;
+using System.Net;
+using Newtonsoft.Json;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.ImportLists.Exceptions;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.Simkl
+{
+    public class SimklParser : IParseImportListResponse
+    {
+        private ImportListResponse _importResponse;
+
+        public SimklParser()
+        {
+        }
+
+        public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse importResponse)
+        {
+            _importResponse = importResponse;
+
+            var series = new List<ImportListItemInfo>();
+
+            if (!PreProcess(_importResponse))
+            {
+                return series;
+            }
+
+            var jsonResponse = JsonConvert.DeserializeObject<SimklResponse>(_importResponse.Content);
+
+            // no movies were return
```

Comment:
`No series were returned`

Reply:
Don't go look at the Trakt implementation :D